### PR TITLE
 LIDO-3543 SEARCH

### PR DESCRIPTION
**Pull request by marhong2**
LIDO-3543
allows for Search to return arrays

### CODE REVIEW

File: formulas/functions/text.js [hunk 1]
```diff
@@ -284,19 +296,13 @@ const TextFunctions = {
...
-      ? WildCard.toRegex(findText, "i")
-      : findText;
-    const res = withinText.slice(startNum - 1).search(findTextRegex);
-    if (res === -1) throw FormulaError.VALUE;
-    return res + startNum;
+  SEARCH: (searchText, searchedInText, startingNum) => {
+    const findText = H.accept(searchText, Types.STRING);
+    const withinText = H.accept(searchedInText, null, undefined, false);
+    const startNum = H.accept(startingNum, Types.NUMBER, 1);
+    if(Array.isArray(withinText))
```
Comment (tyleradams) [inline formulas/functions/text.js:303]: Exercise for you, based on this comment, fix SEARCH
https://github.com/Lido-App/fast-formula-parser/pull/70/files#r786202074

**Review by tyleradams — CHANGES_REQUESTED** [review]
.

File: formulas/functions/text.js (hunk 1 above)
Comment (marhong2) [reply formulas/functions/text.js:303]: I changed this fyi, Git issues

**Review by tyleradams — APPROVED** [review]
Very nice

**Pull request merged**